jira_docugen.tex

```diff
@@ -1,67 +1,55 @@
 \section{Test Cases Summary}\label{test-cases-summary}
 
+Follows the list of test cases documented in this specification.
```

Comment:
“The list of test cases documented in this test specification follows”.

But consider whether this text is necessary at all — I think it's self-evident what is being displayed.